cpp/include/mcts/Node.hpp

```diff
@@ -138,7 +139,7 @@ class Node {
     node_pool_index_t child_index = -1;
     core::action_t action = -1;
     int E = 0;  // real or virtual count
-    float raw_policy_prior = 0;
+    float base_prob = 0; // used for both raw policy prior and chance node probability
     float adjusted_policy_prior = 0;
```

Comment:
Rename this to `adjusted_base_prob` to maintain parallelism. Can add a comment that this is equal to `base_prob`, with potential adjustments from Dirichlet-noise and softmax-temperature.